### DISCUSSION TITLE
Minikube's kubernetes-cli dependency

### DISCUSSION THREAD

**Discussion by tony-sol** [Homebrew/discussions#3284] [Everyday usage] [answered]
How to make brew deal with kubectl installed via Docker Desktop and stop complain about installing kubernetes-cli?
It seems legit to check dependencies by system-wide utils, which already installed but not from brew.

Kubectl itself has dependencies, which could be satisfied by default sys packages - bash e.g. 
Coreutils as kubectl dep not only has another dep which is gmp, but contains common sys utils, which also could be satisfied by default.

So why kubectl is still minikube's dependency?

**Comment by SMillerDev** [comment, marked as answer]
> It seems legit to check dependencies by system-wide utils, which already installed but not from brew.

It might seem that way, but it breaks very often and there is no one way to check if a tool exists.

> So why kubectl is still minikube's dependency?

Because anyone who installs `minikube` needs a `kubectl` and `kubernetes-cli` is the version of `kubectl` that is shipped in Homebrew.
